### PR TITLE
Fixed crash log location for OS X, fixes #1745

### LINKED ISSUE #1745: CONTRIBUTING.md OS X log file
CONTRIBUTING.md:
"Attach Tribler's log file found in %APPDATA% on Windows or on ~/.Tribler/ on Linux/OSX..."

> ls -1 ~/.Tribler/ | grep log
pymdht.log
> file ~/.Tribler/pymdht.log 
~/.Tribler/pymdht.log: empty
> grep -i tribler /private/var/log/system.log | wc -l
 26

Please update CONTRIBUTING.md to point tot the correct log file.

### PR DESCRIPTION
**Pull request by devos50**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -22,8 +22,7 @@ To revert back to your original version of Tribler, [download the installer agai
     * The version of Tribler that you are using, if you are running from a branch, branch name and commit ID.
     * The OS and version you are running.
     * Step by step instructions to reproduce the issue in case it's a bug.
-    * Attach Tribler's log file found in `%APPDATA%` on Windows or on
-      `~/.Tribler/` on Linux/OSX (have a look at its contents before posting it
+    * Attach Tribler's log file found in `%APPDATA%` on Windows, `~/.Tribler/` on Linux or the crash log found in `~/Library/Logs/DiagnosticReports` on OS X (have a look at its contents before posting it
```
Comment (whirm) [inline CONTRIBUTING.md]: Is this better than `syslog -C |grep -i tribler > ~/tribler.log`? or maybe they contain complimentary information?

Comment (devos50) [inline CONTRIBUTING.md]: I just created a crash in the Python interpreter and the full info is written to the files found in `~/Library/Logs/DiagnosticReports`. For each crash, a file is generated with the stack trace. The name of the file indicates the program that has crashed (for instance, I have a `Tribler_2016-01-10-205136_MacBook-Pro-van-Martijn.crash` in this directory).

When I created the crash, the syslog only contained a line like `crash log written to ...` where `...` is the location of the crash log file.

Comment (whirm) [inline CONTRIBUTING.md]: But it contains normal python log entries and stout/err etc, right? Maybe we should add both to the docs.

**Comment by whirm** [comment]
retest this please

**Comment by whirm** [comment]
Oops wong PR

File: CONTRIBUTING.md (hunk 1 above)
Comment (devos50) [inline CONTRIBUTING.md]: I'm not really sure where stdout/stderr is going on OS X, do you know that?

Comment (whirm) [inline CONTRIBUTING.md]: I'd say syslog, that's easy to verify though.

Comment (devos50) [inline CONTRIBUTING.md]: I've started Tribler some minutes ago and I see zero entries regarding stdout/stderr of Tribler in `/var/log/system.log`.

Comment (whirm) [inline CONTRIBUTING.md]: Just modify logger.conf so you get debugging output.

**Comment by devos50** [comment]
@whirm I've added the command you supposed to make it easier for users to extract Tribler-related lines from the syslog.

**Comment by devos50** [comment]
retest this please

**Comment by devos50** [comment]
retest this please

**Pull request merged**